anvilPoll2024MainAnalysis.Rmd

```diff
@@ -219,9 +223,13 @@ ggplot(experienceDf, aes(x=experienceLevel,y=n, fill = experienceLevel)) +
 ggsave(here("plots/researchExperienceLevel_sequentialColor_noUserTypeSplit.png")) #set plot size
 ```
 
+## Experience: General interest in controlled access datasets
+
+**Takeaway:** Over half of respondents report they are extremely interested in working with controlled access datasets.
+
 ## Experience: Controlled Access Datasets
 
-**Takeaway:**
+**Takeaway:** Of the survey provided choices, respondents have accessed or are particularly interested in accessing [All of Us](https://www.researchallofus.org/), [UK Biobank](https://www.ukbiobank.ac.uk/enable-your-research/about-our-data), and [GTEx](https://anvilproject.org/data/consortia/GTEx) (though All of Us and UK Biobank are not currently AnVIL hosted). 2 respondents (moderately or extremely experienced with genomic data) specifically wrote in ["TCGA"](https://www.cancer.gov/ccg/research/genome-sequencing/tcga). The trend of All of Us, UK Biobank, and GTEx being chosen the most is consistent across all 3 research categories (moderately or extremely experienced with clinical, human genomic, or non-human genomic data).
```

Comment:
Let's condense these two H2s to Experience: Controlled Access Datasets.

It could look like:

```
## Experience: Controlled Access Datasets

**Takeaway:** Over half of respondents report they are extremely interested in working with controlled access datasets.

- Of the survey provided choices, respondents have accessed or are particularly interested in accessing [All of Us](https://www.researchallofus.org/), [UK Biobank](https://www.ukbiobank.ac.uk/enable-your-research/about-our-data), and [GTEx](https://anvilproject.org/data/consortia/GTEx) (though All of Us and UK Biobank are not currently AnVIL hosted). 
- 2 respondents (moderately or extremely experienced with genomic data) specifically wrote in ["TCGA"](https://www.cancer.gov/ccg/research/genome-sequencing/tcga). 
- The trend of All of Us, UK Biobank, and GTEx being chosen the most is consistent across all 3 research categories (moderately or extremely experienced with clinical, human genomic, or non-human genomic data).
```